t/plugin/log-rotate.t

```diff
@@ -95,11 +95,89 @@ GET /t
             ngx.say("done")
         }
     }
---- request
-GET /t
 --- response_body
 done
 --- no_error_log
 [error]
 --- error_log
 start xxxxxx
+
+
+
+=== TEST 3: fix: ensure only one timer is running
+--- config
+    location /t {
+        content_by_lua_block {
+            ngx.sleep(0.5)
+            local t = require("lib.test_admin").test
+            local code, _, org_body = t('/apisix/admin/plugins/reload',
+                                        ngx.HTTP_PUT)
```

Comment:
For me, I think this line of code is not very long, we can write it on the same line, so it looks clearer. What do you think？

Reply:
@Firstsawyou 
No. I like the indentation now, which I can see it is a PUT request at the first glance.
We make a new line not just because the old one is too long, but also make the important part of code easier to notice.

Reply:
Okay, got it.